packages/logo/src/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Box } from '@ag.ds-next/box';
+import { AWELogo } from './index';
+
+export default {
+	title: 'layout/AWELogo',
+	component: AWELogo,
+	// subcomponents: [AWELogoContainer],
```

Comment:
I don't know why I thought this was an array. It definitely is not.

```suggestion
	subcomponents: { AWELogoContainer },
```